ddtrace/internal/ci_visibility/recorder.py

```diff
@@ -38,6 +41,9 @@ def __init__(self, tracer=None, config=None, service=None):
         super(CIVisibility, self).__init__()
 
         self.tracer = tracer or ddtrace.tracer
+        if ddconfig.ci_visibility_agentless_enabled and not isinstance(self.tracer._writer, CIVisibilityWriter):
+            writer = CIVisibilityWriter()
```

Comment:
Does the CIVisitilityWriter support the [ddtrace-patch-all](https://ddtrace.readthedocs.io/en/stable/integrations.html#pytest]) option? (I am not 100% sure what it does but I assume it submits non ci app spans)

Reply:
As far as I can tell that option operates at the level of the pytest integration and the writer doesn't need to know about it.